sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleEditForm.java

```diff
@@ -64,16 +70,35 @@ protected void addFields() {
 
 		addValueChangeListener(e -> {
 			defaultValueChangeListener();
-			if (FacadeProvider.getPathogenTestFacade().hasPathogenTest(getValue().toReference())) {
-				getField(SampleDto.PATHOGEN_TEST_RESULT).setRequired(true);
-			} else {
-				getField(SampleDto.PATHOGEN_TEST_RESULT).setEnabled(false);
-			}
+			fillPathogenTestResult();
 		});
 	}
 
-	public void makePathogenTestResultRequired() {
+	public void fillPathogenTestResult() {
 		ComboBox pathogenTestResultField = (ComboBox) getFieldGroup().getField(SampleDto.PATHOGEN_TEST_RESULT);
+
+		boolean hasPathogenTests = FacadeProvider.getPathogenTestFacade().hasPathogenTest(getValue().toReference());
+
+		if (!hasPathogenTests) {
```

Comment:
Remove the two lines that update the enum data and set the pathogen test result value in this if block. "Not done" is meant to point out that there was no pathogen test done for the sample, and that there won't be one in the future as well (i.e. the sample has been dropped without testing it). "Pending" is still meant to be the default when a sample is created and no pathogen test exists yet.

Reply:
In that case the user won't be able to change the status to NOT_DONE if there are no pathogen tests done.
From my understanding of this line, the user should be able to select NOT_DONE if there are no pathogen recorded.
> The NOT DONE laboratory result would ONLY be available for selection when the pathogen test result is on PENDING or there is none recorded (NA), any other pathogen test result would work as is implemented currently.

I could remove the line which sets the default value to NOT_DONE if no pathogen test is recorded, but I would suggest to keep then the approach which updates the enum data set to let the user choose between PENDING and NOT_DONE when no pathogen test was recorded.

Reply:
Maybe I misunderstood the code then. :) The way you described it makes sense to me! So "pending" should still be the default, but adding "not done" to the list is of course necessary, and also to make it editable even if no pathogen test has been recorded yet.